mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -83,6 +83,30 @@ instances:
     #   cert: <CERT_FILE_PATH>
     #   ca: <CA_PATH_FILE>
 
+    ## @param additional_status - list of mappings
```

Comment:
This file is autogenerated. Changes here need to come from the [spec file ](https://github.com/DataDog/integrations-core/blob/master/mysql/assets/configuration/spec.yaml) and then synced with `ddev validate config -s`